src/vivarium_conic_lsff/components/fortification/folic_acid.py

```diff
@@ -197,7 +222,7 @@ def adjust_birth_weights(self, index, birth_weights):
         fortified_status = self.population_view.get(index)[self._fortified]
         idx_fortified = fortified_status.loc[fortified_status=='covered'].index
         if len(idx_fortified):
-            elemental_iron = self.population_view.get(index)[self._iron_consumed]
+            elemental_iron = self.population_view.get(index)[self._iron_consumed].loc[idx_fortified]
```

Comment:
What's going on in this method is definitely not right, but address in a new PR.